pkg/handlers/handlers.go

```diff
@@ -153,14 +153,14 @@ func newFileReader(r io.Reader, options ...readerOption) (fReader fileReader, er
 		return fReader, nil
 	}
 
-	var format archiver.Format
-	format, _, err = archiver.Identify("", fReader)
+	var format archives.Format
+	format, _, err = archives.Identify(context.Background(), "", fReader)
```

Comment:
Can you make this a `context.TODO()` and create a scanning board ticket to clean it up? Thanks!